doc/introduction.rst

```diff
@@ -248,10 +252,10 @@ called. But let's change ``set_alien_hurt()`` to use the clock, so that the
     def set_alien_hurt():
         alien.image = 'alien_hurt'
         sounds.eep.play()
-        clock.schedule_unique(set_alien_normal, 1.0)
+        clock.schedule_unique(set_alien_normal, 1)
```

Comment:
Let's change this to 0.5 as it suggests in #29. 1 will work, of course, but time is a floating point quantity :)